src/components/Header.js

```diff
@@ -39,6 +40,29 @@ export default class Header extends Component {
     );
   }
 
+  renderShelterNavItems() {
```

Comment:
I think it would be a good idea to still have a link to '/pets' for shelters, even if they don't really look at it that often. They may want to make sure a pet is showing up in the general search as a sanity check

Reply:
Good idea